src/decimal-field.tsx

```diff
@@ -48,19 +56,44 @@ export const DecimalInput: React.FC<DecimalInputProps> = ({inputRef, value, defa
         return parts.shift() + localDecimalChar + parts.join('');
     };
 
+    const replace = (value: any) => String(value ?? '').replace(/[.,]/g, localDecimalChar); // use local decimal char
```

Comment:
replace the localDecimalChar in a `replace` input for rifm to prevent the caret from being kept in the position before the delimiter: https://github.com/realadvisor/rifm/issues/65#issuecomment-503073678